mne/viz/tests/test_utils.py

```diff
@@ -117,4 +118,15 @@ def test_auto_scale():
                   dict(grad='auto'), epochs)
 
 
+def test_validate_if_list_of_axes():
+    import matplotlib.pyplot as plt
+    fig, ax = plt.subplots(2, 2)
+    ax = ax.ravel().tolist()
```

Comment:
Otherwise I get:
> `axes must be a list of matplotlib axes objects, got <class 'numpy.ndarray'> instead.`

I could change `_validate_if_list_of_axes` to accept array-like if you want.